tests/functional/lifecycle/LifecycleTask.js

```diff
@@ -915,59 +915,6 @@ describe('lifecycle task functional tests', function dF() {
             });
         });
 
-        it('should expire a version or delete marker in a versioning ' +
```

Comment:
Can you test that versions/delete markers are NOT being expired in a versioning suspended bucket? Or is that covered in like e2e tests?

Reply:
yup for sure, will add these tests soon!